### ISSUE TITLE
how to filter if filter as OR across two or more fields for same model?

### ISSUE THREAD

**Issue by simkimsia** [AltSchool/dynamic-rest#253]
Let’s say I want to run a filter like this
```
User.objects.filter(Q(username="admin") | Q(first_name="test"))
```

I understand drest does not support this in the end point.

How can I modify so as to allow something like this in the endpoint? I am okay to make a PR for this. I just want to get some guidance so I can conform to drest architecture

**Comment by vmihalachi** [comment]
I need this too :)

**Comment by vmihalachi** [comment]
You are right that it can be tricky but, sometimes, you just want to search for a text that can be contained in multiple fields of a model (like in the Django admin page). In those cases they are all ORs

**Comment by ryochiji** [comment]
@vmihalachi - Yep, totally agree this would be useful -- it's just that our team probably won't be able to add this feature anytime soon. If someone wants to contribute a patch to add it, I'm open to it.

**Comment by denny64** [comment]
@ryochiji Do you know if dynamic-rest will work with django-filters to solve this issue? I haven't been able to get them to play nicely together
